src/OrchardCore/OrchardCore.Abstractions/Extensions/JsonSerializerOptionsConfiguration.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.Json.Serialization.Metadata;
+using Microsoft.Extensions.Options;
+using OrchardCore.Json;
+using OrchardCore.Json.Serialization;
+
+namespace OrchardCore.Extensions;
+
+public class JsonSerializerOptionsConfiguration : IConfigureOptions<JsonSerializerOptions>
+{
+    private readonly IEnumerable<IJsonTypeInfoResolver> _typeInfoResolvers;
+    private readonly IOptions<JsonDerivedTypesOptions> _derivedTypesOptions;
+
+    public JsonSerializerOptionsConfiguration(
+        IEnumerable<IJsonTypeInfoResolver> typeInfoResolvers,
```

Comment:
Remove IEnumerable<IJsonTypeInfoResolver> typeInfoResolvers

and any DI resolution of  IEnumerable<IJsonTypeInfoResolver> since users can provide custom Configure<> calls if they want to add custom ones to options.TypeInfoResolverChain

Reply:
done.